src/videojs-http-streaming.js

```diff
@@ -127,47 +127,64 @@ Vhs.canPlaySource = function() {
     'your player\'s techOrder.');
 };
 
-const emeKeySystems = (keySystemOptions, videoPlaylist, audioPlaylist) => {
+const emeKeySystems = (keySystemOptions, mainPlaylist, audioPlaylist) => {
   if (!keySystemOptions) {
     return keySystemOptions;
   }
 
-  const codecs = {
-    video: videoPlaylist && videoPlaylist.attributes && videoPlaylist.attributes.CODECS,
-    audio: audioPlaylist && audioPlaylist.attributes && audioPlaylist.attributes.CODECS
-  };
+  const codecs = {};
 
-  if (!codecs.audio && codecs.video && codecs.video.split(',').length > 1) {
-    codecs.video.split(',').forEach(function(codec) {
-      codec = codec.trim();
+  if (mainPlaylist && mainPlaylist.attributes && mainPlaylist.attributes.CODECS) {
```

Comment:
Looking into it, there is one slight difference in that `codecsForPlaylist` would grab the default audio codec, whereas this should grab the currently used audio playlist's codec. They should, in theory, be the same (or at least compatible), so it may not be a big issue, but in practice who knows what people do. I can see either side though. What do you think?

Reply:
I think we might need to use `codecsForPlaylist` for the edge case described here: https://github.com/videojs/http-streaming/blob/main/src/util/codecs.js#L116

Reply:
That should be covered by using the audio playlist below, since that playlist will have the attributes on it and will overwrite any audio codecs (or overwrite a missing codec if the video rendition doesn't list audio). And in the event that a non default alternate audio playlist is selected, we'd want to use the attributes there over the default one used by codecsForPlaylist

Reply:
:shipit: